src/style.css

```diff
@@ -0,0 +1,90 @@
+@font-face {
+  src: url(fonts/Avenir-Book.ttf);
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: 400;
+}
+
+* {
+  box-sizing: border-box;
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-weight: 300;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+}
+
+.search-string {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  position: relative;
+  width: 100%;
+  margin: 20px 0;
+}
+
+.search-string:hover {
+  border: 1px solid #e1e7ed;
```

Comment:
don't duplicate styles. you can remove it. same for focus

Reply:
done